lib/utils.js

```diff
@@ -44,7 +44,7 @@ function processEnv (config) {
   config.ci.buildNumber = process.env[config.ci.env.buildNumber]
   config.prNumber = process.env[config.ci.env.pr]
   config.isPr = config.prNumber !== 'false'
-  config.branch = process.env[config.ci.env.branch]
+  config.branch = process.env[config.ci.env.branch] || 'master'
```

Comment:
Didn't catch this in the previous PR's. Thanks for fixing this before I even knew it was an issue.